Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule_test.py

```diff
@@ -685,8 +706,10 @@ def test_parse_autonomous_system_indicator(self, taxii_2_client):
                 }
             }
         ]
-
         assert taxii_2_client.parse_sco_autonomous_system_indicator(autonomous_system_obj) == xsoar_expected_response
+        taxii_2_client.update_custom_fields = True
+        assert taxii_2_client.parse_sco_autonomous_system_indicator(autonomous_system_obj) == xsoar_expected_response_with_update_custom_fields
+        taxii_2_client.update_custom_fields = False
```

Comment:
What am I missing here? You are setting the `update_custom_fields` value AFTER calling the `parse_sco_autonomous_system_indicator` function and performing the assertion, right? How does it affect the behavior then?